doc/changelog/03-notations/14819-rm_deprecated_813.rst

```diff
@@ -0,0 +1,32 @@
+- **Removed:**
+  Removed the ``Numeral Notation`` command that was renamed to :cmd:`Number Notation` in 8.13.
```

Comment:
Removed is already part of the sentence, no need to repeat it.

Reply:
Thanks (I had vague remembrance of a previous discussion a while ago on the topic that ended up to the conclusion that the word should be duplicated).